packages/web/src/components/comments/CommentSectionMobile.tsx

```diff
@@ -0,0 +1,96 @@
+import { useGetTrackById } from '@audius/common/api'
+import { useCurrentCommentSection } from '@audius/common/context'
+import {
+  Flex,
+  IconCaretRight,
+  Paper,
+  PlainButton,
+  Skeleton,
+  Text
+} from '@audius/harmony'
+import { Link } from 'react-router-dom'
+
+import { CommentBlock } from './CommentBlock'
+import { CommentForm } from './CommentForm'
+
+const messages = {
+  noComments: 'Nothing here yet',
+  viewAll: 'View All'
+}
+
+const CommentSectionHeader = () => {
+  const {
+    entityId,
+    commentSectionLoading: isLoading,
+    comments
+  } = useCurrentCommentSection()
+
+  const { data: track } = useGetTrackById({ id: entityId })
+
+  const isShowingComments = !isLoading && comments?.length
+
+  return (
+    <Flex
+      direction='row'
+      w='100%'
+      justifyContent='space-between'
+      alignItems='center'
+    >
+      <Text variant='title' size='m'>
+        Comments
+        {isShowingComments ? (
+          <Text color='subdued'>&nbsp;({comments.length})</Text>
+        ) : null}
+      </Text>
+      {isShowingComments ? (
+        <PlainButton iconRight={IconCaretRight} variant='subdued' asChild>
+          <Link to={`${track?.permalink}/comments`}>{messages.viewAll}</Link>
+        </PlainButton>
+      ) : null}
+    </Flex>
+  )
+}
+
+const CommentSectionContent = () => {
+  const {
+    currentUserId,
+    commentSectionLoading: isLoading,
+    comments
+  } = useCurrentCommentSection()
+
+  // Loading state
+  if (isLoading) {
+    return (
+      <Flex direction='row' gap='s' alignItems='center'>
+        <Skeleton w={40} h={40} css={{ borderRadius: 100 }} />
+        <Flex gap='s'>
+          <Skeleton h={20} w={240} />
+          <Skeleton h={20} w={160} />
+        </Flex>
+      </Flex>
+    )
+  }
+
+  // Empty state
+  if (!comments || !comments.length) {
+    return (
```

Comment:
Does it make sense to combine this with the `NoComments` component? If not maybe we move that component inside the desktop file

Reply:
yeah it's a pretty different layout... sure i can move it inside desktop for now, and maybe we have a consolidation run later with mobile-web and desktop as much as seems reasonable

Reply:
Yeah it's a pretty small component anyways and I debated about it having it's own file. 
I think to avoid confusion with mobile would be good to move it away from its own file then 👌